test.js

```diff
@@ -16,7 +16,7 @@ const mapper = ([val, ms]) => delay(ms).then(() => val);
 test('main', async t => {
 	const end = timeSpan();
 	t.deepEqual(await m(input, mapper), [10, 20, 30]);
-	t.true(inRange(end(), 290, 320));
+	t.true(inRange(end(), 290, 700));
```

Comment:
I think the time here has to be changed because the concurrency argument now default to 1, so while before this change the test was running with a concurrency of 3, now is running with a concurrency of 1. On my PC I'm getting a result of 615. I think expected value should be 600.